### ISSUE TITLE
path-for returns empty string instead of "/" for root URLs

### ISSUE THREAD

**Issue by timgilbert** [aroemers/sibiro#2]
Sibiro looks great, thanks!

I'm seeing a problem with matching a route at `"/"`, though:

``` clojure
(require '[sibiro.core :as s])
; => nil
(def r (s/compile-routes #{[:get "/" :root]}))
; => #'user/r
(s/path-for r :root)
; => ""
```

I would expect the root path to come back as `"/"`, which is how I defined it. Using the empty string will probably not do what I want it to if I'm trying to construct a URL out of it.

This is with `[org.clojure/clojure "1.9.0-alpha13"]` and `[functionalbytes/sibiro "0.1.3"]`.

**Comment by aroemers** [comment]
Thank you for the issue report, I will look into this.

**Comment by aroemers** [comment]
@timgilbert, I just released version `0.1.4`, which should fix the issue you are having. Can you verify this?

**Comment by timgilbert** [comment]
Works like a charm @aroemers. Thanks for the quick fix!
